kivy_ios/recipes/sdl3/__init__.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+import sh
+
+from kivy_ios.toolchain import Recipe, shprint
+
+
+class LibSDL3Recipe(Recipe):
+    version = "3.2.8"
+    embed_xcframeworks = ["SDL3"]
+    url = "https://github.com/libsdl-org/SDL/releases/download/release-{version}/SDL3-{version}.tar.gz"
+    include_dir = "include"
+    pbx_frameworks = [
+        "AudioToolbox",
+        "CoreAudio",
+        "CoreFoundation",
+        "CoreHaptics",
+        "CoreVideo",
+        "GameController",
+        "IOKit",
+        "Metal",
+        "QuartzCore",
+        "UniformTypeIdentifiers",
+    ]
+
+    def prebuild_platform(self, plat):
+        if self.has_marker("patched"):
+            return
+        self.apply_patch("uikit-transparent.patch")
+        # self.apply_patch("disable-hidapi.patch")
```

Comment:
Is it some leftover from previous debugging session?

Reply:
`self.apply_patch("disable-hidapi.patch")` is something we have right now in SDL2 to avoid adding an extra permission (during App Store submission).

Not sure if it's still a thing in SDL3. I guess we will found out once we start using SDL3 in production apps.

So, more like a reminder for now. Yeah, I agree about a common recipe in future.

Reply:
Thanks for the context, today I learned :)